### PR TITLE
start? should check enabled?

### LINKED ISSUE #247: Disabling Bullet for a controller action in README does not currently work
The README says that it is possible to disable Bullet for a particular action by using this:

```ruby
class ApplicationController < ActionController::Base
  around_action :skip_bullet

  def skip_bullet
    Bullet.enable = false
    yield
  ensure
    Bullet.enable = true
  end
end
```

I've tried with Rails 4.2, and because the request starts being profiled at the Rack level, `Bullet.enable` has no effect.

### PR DESCRIPTION
**Pull request by lowjoel**
Fixes #247.

I'm not sure if this is the best way to do it -- it seems every detector checks start?

### CODE REVIEW

File: spec/bullet_spec.rb [hunk 1]
```diff
@@ -38,4 +38,22 @@
...
+        ensure
+          Bullet.enable = old_enabled
+        end
+      end
+
+      it 'should not be started' do
+        expect(Bullet.start?).to be(false)
+      end
+    end
+  end
```
Comment (flyerhzm) [inline spec/bullet_spec.rb]: test looks too complicated

Comment (lowjoel) [inline spec/bullet_spec.rb]: there's no way to guarantee that the state after the test would be the same as the test before disabling Bullet, i.e. the other specs may nondeterministically fail.

Comment (flyerhzm) [inline spec/bullet_spec.rb]: @lowjoel it would be better to move around block to `spec/spec_helper.rb`

Comment (lowjoel) [inline spec/bullet_spec.rb]: but this would disable Bullet for every single spec!

Unless we define a without_bullet helper method, then that would work.

**Comment by lowjoel** [comment]
@flyerhzm I added another method to the Bullet module, see if that's better?

File: spec/bullet_spec.rb (hunk 1 above)
Comment (flyerhzm) [inline spec/bullet_spec.rb]: I mean you can add the following code to spec_helper

```
config.before :each do
  Bullet.enable = true
end
```

then in you test you can say 

```
Bullet.enable = false
expect(Bullet.start?).to be(false)
```

a bit cleaner

Comment (lowjoel) [inline spec/bullet_spec.rb]: would you prefer the without_bullet?

Comment (flyerhzm) [inline spec/bullet_spec.rb]: I don't think we need `without_bullet`

btw, test code can be changed to `expect(Bullet).not_to be_start`

Comment (lowjoel) [inline spec/bullet_spec.rb]: Done, have a look.

File: spec/bullet_spec.rb [hunk 2]
```diff
@@ -38,4 +38,17 @@
       end
     end
   end
+
+  describe '#start?' do
+    context 'when bullet is disabled' do
+      before(:each) do |example|
+        Bullet.enable = false
+        example.run
+      end
```
Comment (flyerhzm) [inline spec/bullet_spec.rb]: `example.run` is unnecessary here

Comment (lowjoel) [inline spec/bullet_spec.rb]: woops, yup.

**Comment by flyerhzm** [comment]
thanks

**Comment by lowjoel** [comment]
thanks!

**Pull request merged**